awx/api/generics.py

```diff
@@ -97,6 +103,15 @@ def get(self, request, *args, **kwargs):
         return super(LoggedLoginView, self).get(request, *args, **kwargs)
 
     def post(self, request, *args, **kwargs):
+        if is_proxied_request():
```

Comment:
Interesting. I didn't consider NOT allowing users to login that come through the gateway. I like this restriction. But then I worry about what effect it will have on QE things (ATF & tower-qe).

Reply:
We could leave the POST endpoint alone in case anything is using it, I suppose. But I would hope everything is using either basic auth or token auth. :thinking: